_includes/common-alert.html

```diff
@@ -0,0 +1,9 @@
+{% assign lang = page.lang %}
+{% if lang==nil %}{% assign lang="en" %}{% endif %}
+
+<div class="usa-alert usa-alert--{{ include.type }} margin-bottom-4" id="{% if include.redirect == true %}redirect-pending{% endif %}" markdown="0" role="alert">
+    <div class="usa-alert__body">
+      {% if include.title != nil %}<h2 class="usa-alert__heading">{{ include.title }}</h2>{% endif %}
+      <p class="usa-alert__text">{{ include.text | markdownify }}</p>
```

Comment:
Nice! What happens if someone passes in HTML? Does Markdownify yell at them?

Reply:
Ok, hmmm. I'm not sure. I'm worried that someone in the future might try to pass in html and be confused about the error. It probably just needs to be documented. Are we enforcing this anywhere else in the codebase?

Reply:
Yes it exists in two of the layouts for the page lead and the landing alert component.